tgui/packages/tgui/interfaces/SlotMachine.js

```diff
@@ -50,6 +50,7 @@ const InsertCard = (props, context) => {
 const SlotWindow = (props, context) => {
   const { act, data } = useBackend(context);
   const { scannedCard, money, account_funds, plays } = data;
+  const [wager, setWager] = useSharedState(context, 'wager', 20);
```

Comment:
Rather than using shared state then doing `act('play', { bet: wager })`, should store the `wager` as data (and have an action for changing the wager).